### PR TITLE
add language file for: de

### PR DESCRIPTION
**Pull request by muescha**

### CODE REVIEW

File: src/_locales/de/messages.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{
+  "extName": {
+    "message": "One-Click Erweiterungsmanager"
+  },
+  "extDesc": {
+    "message": "Verwalte einfach deine Erweiterungen: auflisten, enfernen, ein- und auschalten."
```
Comment (fregante) [inline src/_locales/de/messages.json]:
```suggestion
    "message": "Verwalte einfach deine Erweiterungen: auflisten, entfernen, ein- und ausschalten."
```

File: src/_locales/de/messages.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    "message": "Nach Name suchen…"
+  },
+  "gotoOpt": {
+    "message": "Einstellungen…"
+  },
+  "uninstall": {
+    "message": "Erweiterung entfernen…"
+  },
+  "undoInfoMsg": {
+    "message": "Du kannst deine letzte Aktionen rückgäng machen oder wiederherstellen. Dies funtioniert aber nur solange wie dieser Popup-Dialog geöffnet ist. Drücke dazu <kbd>Strg+z</kbd> unter Windows und <kbd>cmd+z</kbd> unter macOS, <kbd>super+z</kbd> unter Ubuntu."
```
Comment (fregante) [inline src/_locales/de/messages.json]:
```suggestion
    "message": "Du kannst deine letzte Aktionen rückgäng machen oder wiederherstellen. Dies funktioniert aber nur solange wie dieser Popup-Dialog geöffnet ist. Drücke dazu <kbd>Strg+z</kbd> unter Windows und <kbd>cmd+z</kbd> unter macOS, <kbd>super+z</kbd> unter Ubuntu."
```

**Comment by muescha** [comment]
oh yes - i should proof read twice if i write something in the night ;)

**Comment by fregante** [comment]
Thank you!

**Comment by HangYang** [comment]
thanks @muescha @fregante

**Pull request merged**